src/framework/piral-core/src/modules/element.ts

```diff
@@ -187,6 +194,47 @@ if (typeof window !== 'undefined' && 'customElements' in window) {
   }
 
   customElements.define(slotName, PiralSlot);
+
+  /**
+   * This is a virtual element to render children defined in React / by Piral in other
+   * frameworks.
+   *
+   * Usage:
+   *
+   * ```
+   * <piral-content cid="123"></piral-content>
+   * ```
```

Comment:
In that spirit, we could mention assignContent here, so that one knows how content is registered when looking at this class for the first time.